src/annotator/components/AdderToolbar.js

```diff
@@ -19,16 +19,16 @@ function ToolbarButton({ badgeCount, icon, label, onClick, shortcut }) {
 
   return (
     <button
-      className="annotator-adder-button"
+      className="AdderToolbar__button"
       onClick={onClick}
       aria-label={title}
       title={title}
     >
-      {icon && <SvgIcon name={icon} className="annotator-adder-button__icon" />}
+      {icon && <SvgIcon name={icon} className="AdderToolbar__icon" />}
       {typeof badgeCount === 'number' && (
-        <span className="annotator-adder-button__badge">{badgeCount}</span>
+        <span className="AdderToolbar__badge">{badgeCount}</span>
       )}
-      <span className="annotator-adder-button__label">{label}</span>
+      <span className="AdderToolbar__label">{label}</span>
```

Comment:
These classes are being used inside a helper component, but I'm using the name of the main component in the file for the class name. This makes it easy for a reader to know where to find the definition of this class if they see it in the DOM. It also avoids having to rename CSS classes if we decide to refactor the internal component structure of a module.